src/containers/ForgotPasswordContainer.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Redirect } from 'react-router-dom';
+
+import ForgotPasswordForm from '../components/ForgotPasswordForm';
+import { sendLink } from '../actions/password';
+
+const forgPassword = (props) => {
+  const {
+    loading,
+    error,
+    isAuthenticated,
+    onSignup,
+    passwordResetToken
+  } = props;
+
+  const submitHandler = ({ email, url }) => {
+    onSignup(email, url);
+  };
+
+  return !passwordResetToken ? (
+    <div>
+      <div>
+        <ForgotPasswordForm
+          authError={error}
+          signedUp={isAuthenticated}
+          loading={loading}
+          submit={submitHandler}
+        />
+      </div>
+    </div>
+  ) : (
+    <Redirect to="/forgotPasswordo" />
```

Comment:
Kindly update route to remove the trailing `o`

Reply:
Done, thank you.